games/static/games/proto.js

```diff
@@ -9,50 +9,52 @@ class Game {
   }
   start() {
     let gameId = 0;
-    let cleanId = 0;
-    let start
-    let restart
-    let board = new Board(this.grid, BOARD_WIDTH, this.setup);
-    this.startButton.addEventListener("mousedown", start = () => {
+    let startButtonEvent = () => {
       if (gameId) {
         clearInterval(gameId);
         gameId = null;
         this.gamePause();
-      } else {
-          this.gameRun();
-          gameId = setInterval(() => {
-            if (board.noAliensLeft()){
-              this.grid.innerHTML = '';
-              this.grid.classList.add('youwin');
-              clearInterval(gameId);
-              this.gamePause();
-            } 
-            board.moveAliens();
-          }, 500);
-      }
-    });
+        return 0
+      } 
+      this.gameRun();
+      gameId = setInterval(() => {
+        if (board.isAlienFree   ()){
```

Comment:
hehe you're funny,
so you didnt use the no word but it still means no aliens on the board
i meant change it to its positive form - board.containsAliens() 
so you can negate it with `!` (this is what other developers expect to see)